knowledge-base/chart-localized-labels.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Localize numeric labels in the Chart
+description: How to localize numeric labels in the Chart
+type: how-to
+page_title: Localize numeric labels in the Chart
+slug: chart-kb-localized-numeric-labels
+position:
+tags:
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Chart for Blazor</td>
+		</tr>
+	</tbody>
+</table>
+
+
+## Description
+
+I am having a Chart and I would like my labels to be localization aware.
+
+
+## Solution
+
+If you require your label to be a number only, you can use the [`Format`]({%slug components/chart/label-template-format%}#format-strings) parameter of the `<ChartSeriesLabels>` tag. It takes a [standard numeric format strings](https://docs.microsoft.com/en-us/dotnet/standard/base-types/standard-numeric-format-strings) and is localization aware.
+
+If you want to expose more information beyond a number, for example, some text, you can utilize the [`Template`]({%slug components/chart/label-template-format%}#templates) parameter, which is exposed from the `<ChartSeriesLabels>` tag. In a JavaScript file implement the desired number formatting by using the `toLocaleString` JavaScript method. An exemplary implementation can be seen in the example below:
+
+### Step by step explanation
```

Comment:
I recommend putting each solution in its own section and list the sections before that, something like https://docs.telerik.com/blazor-ui/knowledge-base/grid-force-refresh